tasks/main.yml

```diff
@@ -147,6 +147,12 @@
             severity: error
             cluster: "{{ cluster_info.ovirt_clusters[0].id }}"
 
+        - name: Update job failed
```

Comment:
Why do we need that? If upgrade fails, then engine should report to audit_log the reason

Reply:
OK, discussed f2f